jade-chrome.el

```diff
@@ -35,10 +35,33 @@
 
 (require 'jade-webkit)
 
+(defvar jade-chrome-bin "chromium")
+(defvar jade-chrome-port "9222")
```

Comment:
The best would be to have a "jade-chrome" custom group as a sub-group of the "jade" group.